app/models/manageiq/providers/redfish/physical_infra_manager/physical_server/operations.rb

```diff
@@ -0,0 +1,62 @@
+require "set"
+
+module ManageIQ::Providers::Redfish
+  module PhysicalInfraManager::PhysicalServer::Operations
```

Comment:
Since this is all Provisioning logic I would expect the module (and file name) to be `provisioning` and not `operations`.